### PR TITLE
[ONNX] Use onnxruntime to run fx tests

### PR DESCRIPTION
**Pull request by justinchuby**
- Enable the mnist test
- Removed `max_pool2d` in the test because we don't have the op yet.
- Add aten::convolution
- Bump onnxscript version

### CODE REVIEW

**Review by BowenBao — APPROVED** [review]

**Review by wschin — APPROVED** [review]
Thanks. LGTM.

**Review by titaiwangms — APPROVED** [review]

**Review by BowenBao — CHANGES_REQUESTED** [review]
linter and onnx test failing

**Review by BowenBao — APPROVED** [review]

File: test/onnx/test_fx_to_onnx_with_onnxruntime.py [hunk 1]
```diff
@@ -54,7 +54,8 @@ def _run_test_with_fx_to_onnx_exporter_reference_runtime(
     )
 
     ref_outputs, _ = pytree.tree_flatten(model(*input_args))
-    ort_outputs = _run_onnx_reference_runtime(onnx_model, input_args)
+    onnx.save_model(onnx_model, "mnist.onnx")
```
Comment (justinchuby) [inline test/onnx/test_fx_to_onnx_with_onnxruntime.py:57]:
```suggestion
```

File: test/onnx/test_fx_to_onnx_with_onnxruntime.py [hunk 2]
```diff
@@ -8,6 +8,7 @@
 
 from typing import Any, Callable, Sequence, Tuple, Union
 
+import onnx
```
Comment (justinchuby) [inline test/onnx/test_fx_to_onnx_with_onnxruntime.py:11]:
```suggestion
```

**Pull request closed without merging**